core/src/main/java/org/radargun/stages/test/legacy/LegacyStressor.java

```diff
@@ -87,15 +91,18 @@ private void runInternal() {
                if (ongoingTx != null) {
                   endTransactionAndRegisterStats(null);
                }
-               stats.begin();
-               this.started = started;
                break;
-            }
-            Operation operation = operationSelector.next(random);
-            try {
-               logic.run(operation);
-            } catch (OperationLogic.RequestException e) {
-               // the exception was already logged in makeRequest
+            } else {
```

Comment:
This seems like you do one operation, call `stats.begin()` (before the other threads have started) and then just busy loops polling stage.isStarted() without running any ops. I don't think this is what you want.